torchattacks/attack.py

```diff
@@ -388,6 +378,63 @@ def to_type(inputs, type):
             raise ValueError(type + " is not a valid type. [Options: float, int]")
         return inputs
 
+    @staticmethod
+    def _save_adv_examples(
```

Comment:
Perhaps adding a type constraint or default parameters value for each parameter here would help other maintainers have a clear definition of parameter types.